performant-software/dm-2
# Add two new highlight colors (light blue and light green)

### What this PR does
- Adds two colors, `cyanA100` and `lightGreenA200`, to highlight options for images/text
- Expands width of color picker to accommodate new colors
- Supports title coloration for highlights of new colors

### Additional notes

- Per https://github.com/performant-software/dm-2/issues/290#issuecomment-878560025, this entire feature appears somewhat inaccessible to colorblind users; some of the existing colors could be changed to improve accessibility in the future


### Status


### Steps to test
- Visit https://dm-2-staging.herokuapp.com/
- Log in
- Open or create a project
- Open or create a document
- Click the color picker tool and ensure that there are 10 colors
- Create highlights/annotations with the new colors:

> ![Screen Shot 2021-07-12 at 4 27 22 PM](https://user-images.githubusercontent.com/7234006/125351562-3b6e0980-e32e-11eb-905f-d468607640b9.png)
> ![Screen Shot 2021-07-12 at 4 27 13 PM](https://user-images.githubusercontent.com/7234006/125351563-3c06a000-e32e-11eb-8bc9-15de9a960834.png)
> 

- Mouse over created highlights/annotations and ensure that the title background is not gray, and matches the color of the highlight/annotation
